styles.css

```diff
@@ -0,0 +1,63 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+}
+
+body {
+    font-family: "Poppins", sans-serif;
+    color: #172B4D;
+    font-size: 16px;
+    
+}
+
+.nav-container {
+    background: #ffffff;
+    height: 50px;
+    max-width: 375px;
+}
+
+.wrapper {
+    max-width: 327px;
+    margin: 0 auto;
+}
+
```

Comment:
- [ ] Please make sure your design looks as close as possible to the figma design.

Figma Design | Your Design
------------- | -------------
![Figma design](https://user-images.githubusercontent.com/74814780/176651460-41572f17-5471-45b5-96b8-88b6ecd0e901.png)  |  ![User design](https://user-images.githubusercontent.com/74814780/176651629-2b372205-913a-4395-b677-2e90118f9613.png)